ui/src/containers/Capabilities/index.js

```diff
@@ -185,15 +189,8 @@ const Launch = ({ browser: { name, version }, history }) => {
                                 name: "Manual session",
                             },
                             capabilities: {
-                                alwaysMatch: {
-                                    browserName: `${name}`,
-                                    browserVersion: `${version}`,
-                                    "selenoid:options": {
-                                        enableVNC: true,
-                                        sessionTimeout: "60m",
-                                        labels: { manual: "true" },
```

Comment:
this is breaking change actually

Reply:
I was following another [documentation]( https://www.w3.org/TR/webdriver1/#capabilities ) and also the `Selenium-Webdriver` [implementation](https://github.com/SeleniumHQ/selenium/blob/bf2012368e5f094dab8b4ed8d2caa87ea2b3434e/javascript/node/selenium-webdriver/lib/webdriver.js#L35-L46). 

An even the mozilla mention the list of as:

* acceptInsecureCerts
* browserName
* browserVersion
* platformName
* pageLoadStrategy
* proxy
* setWindowRect
* timeouts
* unhandledPromptBehavior

Therefore limiting the `capabilities` just to the allowed properties should not brake anything. I test it locally with out issues. Also while adding `firstMatch` or `alwaysMatch` is accepted it does create an issue for `Microsoft Edge`. The `selenoid:options` are still passed, but in the `desiredCapabilities` without having the use the reserve name `selenoid:options`.